app/code/Magento/Checkout/view/frontend/web/js/view/shipping.js

```diff
@@ -247,8 +247,7 @@ define([
          */
         setShippingInformation: function () {
             if (this.validateShippingInformation()) {
-                quote.billingAddress(null);
-                checkoutDataResolver.resolveBillingAddress();
```

Comment:
Is this used anywhere else? Otherwise, the whole function could probably be removed.

Reply:
@josefbehr , I am sorry, I completely missed your comment. So the setShippingInformation is called on clicking the next button in the shipping step of checkout. 

ValidateShippingInformation is called to validate the shipping address and shipping methods. So either of the two methods cannot be removed. Please let me know if you have any concerns on the PR.